doc/api/app.rst

```diff
@@ -22,5 +22,27 @@ This can be done by::
 **Do not instantiate the** :class:`App` **class!** It is designed to be used in a purely static way.
 
 
+Application configuration
+-------------------------
+
+Configuration of the application is saved in the
+:class:`GlobalConfiguration <global_configuration.GlobalConfiguration>` class. This class can be
+created before the :meth:`App.initialize` is called and passed in as a
+parameter. This way the same configuration can be used between re-initialization of the
+application. The configuration can be changed even during the application run easily by setting
```

Comment:
"The configuration can easily be changed, even while an application is running, by setting"